src/main/java/racingcar/view/InputView.java

```diff
@@ -0,0 +1,42 @@
+package racingcar.view;
+
+import java.util.Scanner;
+import racingcar.domain.dto.ParticipantsRequestDto;
+import racingcar.domain.dto.TurnRequestDto;
+
+public class InputView {
+
+    private static Scanner scanner = new Scanner(System.in);
+    private static final String NAME_MESSAGE = "경주할 자동차 이름을 입력하세요(이름은 쉼표(,)를 기준으로 구분).";
+    private static final String TURN_MESSAGE = "시도할 횟수는 몇 회인가요?";
+    private static final String DELIMITER = ",";
+    private static final String ERROR_LOG = "빈 공백은 들어올 수 없습니다.";
+
+    private static String getConsoleTextFrom(final String message) {
+        OutputView.printMessage(message);
+        String line = scanner.nextLine().trim();
+        try {
+            validateInput(line);
+        } catch (IllegalArgumentException e) {
+            e.printStackTrace();
+            return getConsoleTextFrom(message);
+        }
+        return line;
```

Comment:
아래와 같이 return문을 try 문 안에 두는 것이 좋을 것 같아요. 😃 
```suggestion
        OutputView.printMessage(message);
        String line = scanner.nextLine().trim();
        try {
            validateInput(line);
            return line;
        } catch (IllegalArgumentException e) {
            e.printStackTrace();
            return getConsoleTextFrom(message);
        }
```